examples/chat/chat_server/lib/src/generated/channel.dart

```diff
@@ -67,7 +67,7 @@ abstract class Channel extends _i1.TableRow {
   @Deprecated('Will be removed in 2.0.0')
   Map<String, dynamic> toJsonForDatabase() {
     return {
-      'id': id,
+      if (id != null) 'id': id,
```

Comment:
No, not at all. Since a null value is always treated the same as if the key wasn't there it is functionally the same. It will still be an invalid request to send an object to the update method without an id or with an id to the insert method.